packages/deploy-config-generator-shared/src/utils/installed-check.ts

```diff
@@ -0,0 +1,19 @@
+import { findInstalledPackages } from '@sap-ux/nodejs-utils';
+import type { WorkspaceConfiguration } from 'vscode';
+
+/**
+ * Checks if S/4 generator extension is installed.
+ * Note that it may not be loaded by Fiori Generator even if its installed in the case
+ * there are other extensions installed also.
+ *
+ * @param vscWorkspaceConfig
+ * @returns true if the S/4 generator extension is installed
+ */
+export async function isS4Installed(vscWorkspaceConfig?: WorkspaceConfiguration): Promise<boolean> {
+    const s4ExtGenName = '@sapux/s4-fiori-gen-ext';
```

Comment:
Reference to a non-public generator name.

Reply:
See option `additionalPrompts.confirmConfigUpdate` . This will be set by consumer

Reply:
Cool thx